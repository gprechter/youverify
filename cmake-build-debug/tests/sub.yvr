x: INT
y: INT
result: INT
done: BOOL
x = 10
y = 4
result = x - y
done = true
