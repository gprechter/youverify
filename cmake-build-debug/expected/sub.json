{
	"result": 6,
	"done": true,
	"y": 6,
	"x": 0
}
